ietf/home/templates/home/home_page.html

```diff
@@ -5,6 +5,7 @@
 
 {% block main_content %}
 <main role="main">
+    <h1 class="sr-only">Welcome to the IETF (Internet Engineering Task Force)</h1>
```

Comment:
Did you want to do that `abbr` @holloway mentioned? Or is putting the full it in brackets better?

Reply:
I think abbr is good too... if it was for sighted people too then it's probably worth saying both, but as this is for screen readers only I think it's probably better to just use the abbr tag. E.g.

```suggestion
    <h1 class="sr-only">Welcome to the <abbr title="Internet Engineering Task Force">IETF</abbr></h1>
```